packages/x-date-pickers/src/internals/hooks/validation/models.ts

```diff
@@ -24,13 +24,13 @@ export interface BaseTimeValidationProps extends FutureAndPastValidationProps {}
  */
 export interface TimeValidationProps<TDate> {
   /**
-   * Min time acceptable time.
-   * For input validation date part of passed object will be ignored if `disableIgnoringDatePartForTimeValidation` not specified.
+   * Minimal time acceptable time.
+   * The date part of the object will be ignored unless `props.disableIgnoringDatePartForTimeValidation === true`.
    */
   minTime?: TDate;
   /**
    * Max time acceptable time.
```

Comment:
```suggestion
   * Maximal selectable time.
```
Just to align with picker descriptions